torchtune/modules/peft/lora.py

```diff
@@ -16,6 +17,12 @@
 from torchtune.modules.peft import AdapterModule
 
 
+class LoRATrainable(Enum):
```

Comment:
Is now a good time to bring up that this name makes no sense?

It would be better as ``Trainable`` or something, but it's clearly not just ``LoRATrainable``

Reply:
Now is not the time I wanna tackle that tbh. We can put up an issue for a horizontal refactor if it really irks you

Reply:
Please put up an Issue

Reply:
#2636